src/main/java/com/exactpro/th2/hand/services/mstore/TransportMessageStoreSender.java

```diff
@@ -35,11 +35,11 @@ public class TransportMessageStoreSender implements MessageStoreSender<RawMessag
 	private final String book;
 	private final String sessionGroup;
 
-	public TransportMessageStoreSender(CommonFactory factory) {
-		messageRouter = factory.getTransportGroupBatchRouter();
-		CustomConfiguration customConfiguration = factory.getCustomConfiguration(CustomConfiguration.class);
+	public TransportMessageStoreSender(Config config) {
+		messageRouter = config.getFactory().getTransportGroupBatchRouter();
```

Comment:
Could we rename Config to Context because getting factory from config looks strange?